cache/sampleStore.js

```diff
@@ -30,7 +31,7 @@ const constants = {
       'okRange',
       'writers',
     ],
-    sample: ['relatedLinks'],
+    sample: ['relatedLinks', 'user'],
```

Comment:
Originally I thought this test would be in tests/cache/sampleStore.js, but since that test does not setup any user, the samples populated would NOT have the user object.